app.js

```diff
@@ -1,14 +1,25 @@
 const express = require("express");
 const app = express();
 const { getCategories } = require("./controllers/categories.controller");
+const { getReviewById } = require("./controllers/reviews.controller");
+const { handlePSQL400s, handleCustomErrors, handle500Statuses } = require("./error_handling");
+
+app.use(express.json());
 
 app.get("/api", (req, res) => {
     res.status(200).send({ msg: "Server is up and running" });
```

Comment:
This callback function could go into the controller where we deal with the request and response.